index.html

```diff
@@ -175,92 +177,68 @@ <h5 class="letter1">Profesional Art Printing Data</h5>
+            <li class="fontaboutmeskills">Boostrap</li>
+            <li class="fontaboutmeskills">Ruby on rails</li>
+            <li class="fontaboutmeskills">RSpec</li>
+            <li class="fontaboutmeskills">Capybara</li>
+        </ul>
+        <ul class="grid3">
+            <li class="aboutmeskills3">Skils</li>
+            <li class="fontaboutmeskills">Codekit</li>
+            <li class="fontaboutmeskills">Github</li>
+            <li class="fontaboutmeskills">Codepen</li>
+            <li class="fontaboutmeskills">Gitlab</li>
+            <li class="fontaboutmeskills">Gitlab</li>
+        </ul>
     </section>
-    <div class="clear-fix"></div>
-    <!-- Start footer section -->
-    <footer>
-        <section class="wrapper">
-            <h2 class="secondary-header">I'm always interested in hearing about new projects, so if you'd like to chat
-                please get in touch.
-            </h2>
-            <form class="contact-form" action="">
-                <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
-                <input type="email" name="email" id="email" placeholder="Email address" required>
-                <textarea name="message" id="message" rows="7" placeholder="Enter text here" required></textarea>
-                <button type="submit">Get in touch</button>
-            </form>
-            <div class="footer-social-media">
-                <ul class="links">
-                    <li>
-                        <a href="#githublink"><img src="imgs/githubimg.png" alt="githubimage"></a>
-                    </li>
-                    <li>
-                        <a href="#linkeinlink"><img src="imgs/linkeinimg.png" alt="linkedinimage"></a>
-                    </li>
-                    <li>
-                        <a href="#angelistlink"><img src="imgs/Angelistimg.png" alt="angelistimage"></a>
-                    </li>
-                    <li>
-                        <a href="#twitterinlink"><img src="imgs/twitter.png" alt="twitterimg"></a>
-                    </li>
-                    <li>
-                        <a href="#mediumlink"><img src="imgs/Mediumimg.png" alt="mediumimg"></a>
-                    </li>
-                </ul>
-            </div>
-        </section>
-    </footer>
-    <!-- End footer section -->
-</body>
-
-</html>
\ No newline at end of file
+    <footer class="contact">
+        <h2 class="secondary-header">I'm always interested in hearing about new projects, so if you'd like to chat
+            please get in touch.</h2>
+        <form class="contact-form" action="https://formspree.io/f/xayknejl" method="post">
+            <input type="text" maxlength="30" placeholder="Full Name" required>
+            <input type="email" placeholder="Email address" required>
+            <textarea placeholder="Enter text here" rows="7"></textarea>
+            <button type="submit" class="btn">Get in touch</button>
+        </form>
```

Comment:
- [ ] great job implementing the formspree link , when I click on submit the form I get this error : 

![image](https://user-images.githubusercontent.com/89282221/186489112-469a9259-0708-4a1b-9edd-4eedcca0b274.png)

kindly add the `name` attribute for each form element that you have so the data will be successfully submitted